### PR TITLE
post_install_template fix

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
Signed-off-by: Rahul Tripathi <rahul.tripathi@seagate.com>

# Problem Statement
- fix for post_install_teplate not working

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by veerendra-simha-garikipati — APPROVED** [review]
+1

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -198,13 +198,15 @@ def post_install(post_install_template: str):
         Utils._set_to_conf_file('support>local_path', default_sb_path)
         os.makedirs(default_sb_path, exist_ok=True)
 
+        post_install_template_index = "post_install_index"
```
Comment (vastradparayya) [inline py-utils/src/setup/utils.py:201]: This can be single quotes

Comment (rahul-tripathi-git) [reply py-utils/src/setup/utils.py]: Done

**Review by vastradparayya — APPROVED** [review]

**Pull request merged**